src/agent/communicator/src/http_socket.hpp

```diff
@@ -60,67 +59,4 @@ namespace http_client
     private:
         boost::asio::ip::tcp::socket m_socket;
     };
-
-    class HttpsSocket : public IHttpSocket
```

Comment:
These changes could be squashed to the first commit, so the new class is introduced from the beginning in a separate file.

Reply:
Done!